### PR TITLE
Improve check for plugin installation

### PR DESCRIPTION
**Pull request by yakky**

### CODE REVIEW

File: roles/es/tasks/main.yml [hunk 1]
```diff
@@ -33,9 +33,16 @@
...
+  command: /usr/share/elasticsearch/bin/elasticsearch-plugin list
+  when: es == 1
+  register: es_plugins
+  tags:
+     - setup
+
 - name: Install ES plugin
   command: /usr/share/elasticsearch/bin/elasticsearch-plugin install analysis-icu
-  when: es == 1
+  when: es == 1 and 'analysis-icu' in es_plugins
```
Comment (xrmx) [inline roles/es/tasks/main.yml]: not in?

File: roles/app/tasks/main.yml [hunk 2]
```diff
@@ -169,6 +169,7 @@
 - name: run provision_elasticsearch command
   become_user: "{{ rtd_user }}"
   command: "{{ rtd_virtualenv }}/bin/python {{ rtd_root }}/manage.py provision_elasticsearch --settings=readthedocs.settings.managed"
+  ignore_errors: yes
```
Comment (xrmx) [inline roles/app/tasks/main.yml]: This is for the case when no es cluster is available?

Comment (yakky) [reply roles/app/tasks/main.yml]: Yes

File: roles/es/tasks/main.yml (hunk 1 above)
Comment (yakky) [reply roles/es/tasks/main.yml]: Fixed

**Pull request merged**